.github/labeler.yml

```
@@ -0,0 +1,20 @@
+# https://github.com/actions/labeler#common-examples
+# Adapted from https://github.com/rapidsai/dask-cuda/blob/main/.github/CODEOWNERS
+# Labels culled from https://github.com/rapidsai/dask-cuda/labels
+
+Python:
+ - 'dask_cuda/**'
+
+cpp:
+ - 'cpp/**'
+
+CMake:
+ - '**/CMakeLists.txt'
+ - '**/cmake/**'
+
+ci:
```

This repo has no `ci` or `gpuCI` label.

---

```suggestion
gpuCI:
```

I just added the missing label, so we just need to make this change